### DISCUSSION TITLE
Modelleerprincipes - U2. Informatiemodel maakt gebruik van relevante nationale standaarden

### DISCUSSION THREAD

**Discussion by pmaria** [Geonovum/disgeo-imsor#53] [General] [answered]
Modelleerprincipe [[U2]](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#u2-informatiemodel-maakt-gebruik-van-relevante-nationale-standaarden) - De gegevensmodellering zal plaatsvinden conform afspraken tussen de organisaties Kadaster – Geonovum – VNG-Realisatie en conform relevante standaarden voor informatiemodellering.

Discussie uit expertgroepsessie:
> - Michel: Het gaat hierbij ook om internationale standaarden. [voorstel voor aanpassing van de titel]: 'nationale' weglaten, of 'nationale en internationale standaarden' gebruiken.
> - Lennart: Bij 'nationale standaarden' gaat het meestal over een interpretatie/adoptie (ie localised version) van internationale standaarden. Dit is belangrijk aangezien je in de natioanle context concretere afspraken maakt over alle opties die beschreven staan in de internationale versies van de standaarden.
> - Namen van partijen weglaten
> - woordje afspraken weglaten

**Comment by lvdbrink** [comment, marked as answer]
- +1 voor het weglaten van het woord 'nationale'
- +1 voor het weglaten van namen van partijen (ik neem aan dat op dit stukje tekst gedoeld werd: 'de organisaties Kadaster – Geonovum – VNG-Realisatie'
- +1 voor het weglaten van 'afspraken'

**Comment by lvdbrink** [comment]
Bij de discussie in de expertgroepbijeenkomst van 17/6 bleek dat iedereen het wel over dit uitgangspunt eens is. Wel is het belangrijk om te vermelden dat, als iets niet in de (nationale) standaarden staat, dat we het wel proberen terug te brengen naar deze standaarden. Noot 1 dus veralgemeniseren naar alle standaarden.

**Comment by lvdbrink** [comment]
Noot is toegevoegd zoals vermeld in [expertgroep 8 juli](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/08-07-2021-aantekeningen.md).
